src/Inventory.zig

```diff
@@ -1515,6 +1580,10 @@ pub fn deposit(dest: Inventory, destSlot: u32, carried: Inventory, amount: u16)
 	Sync.ClientSide.executeCommand(.{.deposit = .{.dest = .{.inv = dest, .slot = destSlot}, .source = .{.inv = carried, .slot = 0}, .amount = amount}});
 }
 
+pub fn addHealth(health: f32, cause: main.game.DamageType) void {
+	Sync.ClientSide.executeCommand(.{.addHealth = .{.health = health, .cause = cause, .previous = main.game.Player.super.health}});
+}
```

Comment:
Could you move this somewhere else, preferably not in between all the inventory functions?